Sewer56.Patcher.Regravitified.Cli/Commandline.cs

```diff
@@ -10,7 +10,7 @@
 #if SRDX || REGRAV
 using Sewer56.Patcher.Riders.Common.Utility;
 #endif
-#if REGRAV
+#if REGRAV || SRTE
```

Comment:
Doublecheck if this define is needed. Might not be needed and throw a warning.

Reply:
I've included it there because it throws an error due to not having the NKIT utility in the path. But that is due to the lower comment, so I will remove it now.